### PR TITLE
Fix #968 Add reactive locale in links to DocsLink router-link

### LINKED ISSUE #968: DocsLink create router-link without reactive locale in relative links
* **The current implementation**
  - in DocsHelper.Link:
``` import { locale } from '../../../../helpers/I18nHelper'```
``` DocsHelper.link('configurationGuide.readMore', `/${locale}/services/components-config`) ```
  - in DocsLink.vue:
    router-link receive href without reactive locale

* **Proposed implementation**
  - in DocsHelper.Link:
``` DocsHelper.link('configurationGuide.readMore', `/services/components-config`) ```
  - in DocsLink.vue:
    router-link take reactive locale from useI18n hook

### PR DESCRIPTION
**Pull request by RVitaly1978**
## Description

close #968

Some of the changes:
  - add processing locale within DocsLink
  - replace relative links in page-config files
  - rewrite DocsLink in Composition API style

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/src/components/DocsLink.vue [hunk 1]
```diff
@@ -5,39 +5,66 @@
       inline
       tag="span"
       v-if="preText"
-      :value="$te(preText) ? $t(preText) : preText"
+      :value="$t(preText)"
```
Comment (asvae) [inline packages/docs/src/components/DocsLink.vue:8]: Sorry, what was the reasoning here?

Comment (RVitaly1978) [reply packages/docs/src/components/DocsLink.vue:8]: ```$t(path)``` returns 'path' translate (if translation exist in locale) or 'path' itself (if no translation in locale).
Therefore this code ```$te(preText) ? $t(preText) : preText``` is redundant

File: packages/docs/src/locales/ru/ru.json [hunk 2]
```diff
@@ -3156,6 +3159,7 @@
     "title": "Типографика",
     "description": "В Vuestic UI применяются следующие стили типографики.",
     "headings": "Заголовки",
+    "affected": "Применено",
     "display1": "`class=\"display-1\"`",
```
Comment (asvae) [inline packages/docs/src/locales/ru/ru.json:3163]: Pretty sure this shouldn't go to translations.

**Review by asvae — APPROVED** [review]
Feel free to merge after fixes

File: packages/docs/src/components/DocsLink.vue (hunk 1 above)
Comment (asvae) [reply packages/docs/src/components/DocsLink.vue:8]: That's not true, because you also get a warn.

**Pull request merged**